### DISCUSSION TITLE
Add Message Button on User Profile

### DISCUSSION THREAD

**Discussion by jsonpreet** [deso-protocol/dips#45] [Medium CIPs]
I want to submit PR for Message Button in User Profile, with this users can easily send messages to their followers or nonfollowers, it's similar to Twitter.

Process:
When clicking on Message Button, It opens Message Modal.

**Comment by ShhubhamA** [comment]
YES Very much needed

**Reply by tijno** [reply]
Agree 120% need this.

**Comment by lazynina** [comment]
I think this is a good idea and something we've wanted to add but haven't had the time.  Can you provide some mocks up of what you think this will look like?

**Reply by jsonpreet** [reply]
Sure i will show mockup here tomorrow :)
